boards/pulga/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "pulga" if BOARD_PULGA
+
+config BOARD_PULGA
+    bool
+    default y
+    select HAS_PERIPH_I2C
+    select HAS_PERIPH_PWM
+    select HAS_PERIPH_SPI
+    select HAS_PERIPH_UART
+    select CPU_MODEL_NRF52840XXAA
+
+source "$(RIOTBOARD)/common/nrf52xxxdk/Kconfig"
```

Comment:
Is it intentional to keep the `common/nrf52xxxdk` references?

The makefiles seems to refer to the `common/nrf52`

Reply:
Hello @MrKevinWeiss, thank you for the feedback!

I fixed the `Kconfig` definition to point to `common/nrf52`.

I will run the tests and provide the output.

I will also discuss with the team if we could provide a board to a maintainer. Per chance, would there be a maintainer that is based in South America? Our lab is in São Paulo - Brazil.